babel.config.js

```diff
@@ -40,5 +40,14 @@ module.exports = (api) => {
   return {
     ...baseConfig,
     plugins: [...baseConfig.plugins, 'react-native-reanimated/plugin'],
+    env: {
+      // Uncomment if you want to remove all logs in development
+      // development: {
+      //   plugins: ['transform-remove-console'],
+      // },
+      production: {
+        plugins: ['transform-remove-console'],
```

Comment:
Do you know if this conflicts with the plugins above on line 42? Do we need to spread the others on?

Reply:
Seems to be working just fine for me. Going to merge and we can test on TestFlight some more